.infrastructure/helm-chart/todoapp/templates/pv.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{.Chart.Name}}-pv
+  namespace: {{.Values.todoapp.namespace}}
```

Comment:
PersistentVolumes are cluster-wide resources and do not support namespaces. Remove the `namespace` field to avoid configuration errors.